common/djangoapps/student/views.py

```diff
@@ -1617,6 +1617,7 @@ def create_account_with_params(request, params):
             request.session[pipeline.AUTH_ENTRY_KEY] = pipeline.AUTH_ENTRY_REGISTER_API
             pipeline_user = None
             error_message = ""
+            access_token = request.POST["access_token"]
```

Comment:
This line might crash if there is no `access_token` key in POST, so change it to `request.POST.get("access_token")`

Reply:
It's a copy from line 1280